packages/nc-gui/pages/index/[typeOrId]/form/[viewId]/index/index.vue

```diff
@@ -72,14 +72,17 @@ const onDecode = async (scannedCodeValue: string) => {
 <template>
   <div class="h-full flex flex-col items-center">
     <div
-      class="color-transition relative flex flex-col justify-center gap-2 w-full max-w-[max(33%,600px)] m-auto py-4 pb-8 px-16 md:(bg-white dark:bg-slate-700 rounded-lg border-1 border-gray-200 shadow-xl)"
+      class="color-transition flex flex-col justify-center gap-2 w-full max-w-[max(33%,600px)] m-auto py-4 pb-8 px-16 md:(bg-white dark:bg-slate-700 rounded-lg border-1 border-gray-200 shadow-xl)"
     >
       <template v-if="sharedFormView">
-        <h1 class="prose-2xl font-bold self-center my-4" style="word-break: break-all">
+        <h1 class="prose-2xl font-bold self-center my-4 break-words">
```

Comment:
`break-words` gives `overflow-wrap: break-word;`.  It should be `break-all` for `word-break: break-all`.

Ignore this comment if you switch to use `break-word`.